util/pkg/architectures/architectures.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package architectures
+
+import (
+	"fmt"
+	"runtime"
+
+	"k8s.io/klog"
+)
+
+type Architecture string
+
+var (
+	ArchitectureAmd64 Architecture = "amd64"
+)
+
+func FindArchitecture() (Architecture, error) {
+	switch runtime.GOARCH {
+	case "amd64":
+		return ArchitectureAmd64, nil
+	default:
+		return "", fmt.Errorf("unsupported arch: %q", runtime.GOARCH)
+	}
+}
+
+func GetArchitectures() []Architecture {
```

Comment:
`GetSupportedArchitectures()` ?

Reply:
I agree, but as it is not used for now, I just removed it.
I will use the new name when I add the ARM64 pice.